patroni/postgresql/bootstrap.py

```diff
@@ -76,17 +80,21 @@ def error_handler(e):
         ret = self._postgresql.pg_ctl('initdb', *options)
         if pwfile:
             os.remove(pwfile)
-        if not ret:
+        if ret:
```

Comment:
it's a refactoring PR anyway, so what about  renaming  `ret` to `initdb_has_succeded` ?